test/datafmt.jl

```diff
@@ -93,6 +93,14 @@ let x = [1,2,3], y = [4,5,6], io = IOBuffer()
     @test readcsv(io) == [x y]
 end
 
+let x = [0.1 0.3 0.5], io = IOBuffer()
```

Comment:
BTW, you could also put the issue number in a comment, in case somebody wonders why we add the leading `0`.